### DISCUSSION TITLE
OpenRAOD on HPC

### DISCUSSION THREAD

**Discussion by V-Deepak-7** [The-OpenROAD-Project/OpenROAD#5257] [Q&A] [answered]
Hello all, 
I am running a few designs using the Open ROAD flow scripts. The ./setup.sh needs sudo access as I run the designs to get faster results. I use High-Performance Cluster computing at my university, where I do not have sudo privileges. 
Is anybody using it the same way that I am? If yes, how did you solve this?
 It took me a few weeks to install many modules. It was a back-and-forth communication for every missing library in the building. 
Also, where can I find any guide or manual regarding variable names like Place_density or others? Is there anything with the .sdc file writing as well?

**Comment by maliberty** [comment]
Can you use Docker?

**Reply by V-Deepak-7** [reply]
No, but I can use singularity or Apptainer.

**Reply by maliberty** [reply]
We don't support those but you might be able to make them work yourself.  @vvbandeira any suggestions?

**Reply by V-Deepak-7** [reply, marked as answer]
current solution: I requested the HPC team to install all modules separately and I load all modules every time I need to run the ORFS. I was just curious if there was any other way when we don't have the sudo privileges and docker access.  Thank you for your response @maliberty
